### DISCUSSION TITLE
TMS Optimize problem

### DISCUSSION THREAD

**Discussion by VictorS-M** [simnibs/simnibs#440] [8. Simulation workflows] [answered]
Hello to all the wonderful SIMNIBS community!
I'm currently going through a TMS Optimize process, but when Simnibs gives me the results, the coil intersects the head.
Do you know how I can solve this problem ?
Thank you so much.

**Comment by TorgeW** [comment, marked as answer]
The current TMS optimization doesn't take into account coil-head intersections. 
The coil is always placed perpendicular to the head based on the coil center (the center of the coils coordinate system).

Depending on your coil it might be a solution to increase the distance setting. This although might not work well for all coils. 

The next version of Simnibs will contain a new optimization method which takes coil-skin intersections into account. 
We hope to release before the end of the year.

**Reply by VictorS-M** [reply]
Many thanks for your reply. It was really helpful. 
I'll move the coil manually to avoid future intersections.
